src/Generators/ModelGenerator.php

```diff
@@ -53,6 +54,18 @@ protected function populateStub(string $stub, Model $model)
         return $stub;
     }
 
+    private function buildClassPhpDoc(Model $model)
+    {
+        $phpDoc = '/**' . PHP_EOL;
+        /** @var Column $column */
+        foreach ($model->columns() as $column) {
+            $phpDoc.= " * @property {$column->phpDocDataType()} \${$column->name()}" . PHP_EOL;
```

Comment:
This would be lighter weight if the DocBlock data type were inferred here from data type and not added to the Lexer/Column as it has no other value within Blueprint other than this generator.

Reply:
Still would like to see this done. Then I'm glad to merge.

If you don't have time, no worries. I will use the work you have done as a base and finish this up Tuesday.